spec/message_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'message'
+require 'twilio-ruby'
+
+describe Message do
+  it 'receives a text message after order is placed (using VCR)' do
+    VCR.use_cassette('twilio') do
+      subject.send_text('Thanks')
```

Comment:
No expectations in this test!
Injection Twilio, or even stubbing the methods on the Twilio class would have been a simpler approach to testing.